test/gtest/uct/ib/test_dc.cc

```diff
@@ -340,116 +331,39 @@ UCS_TEST_P(test_dc, dcs_ep_flush_pending) {
     EXPECT_EQ(0, iface->tx.stack_top);
 }
 
-/* Check that the following sequnce works ok:
- * - Add some pending request to DCI wait queue
- * - Try to send something from this ep. This will force ep to take free DCI
- *   (the send will not succeed anyway)
- * - Progress all pendings
- * - Make sure that there is no any assertion and everyting is ok
- *   (just send something).
- * */
-UCS_TEST_P(test_dc, dcs_ep_am_pending) {
```

Comment:
why did u remove 2 tests ? (this and dcs_dci_leak)?

Reply:
This one is almost identical to dcs_ep_flush_pending(extended it in the last commit to fully cover dcs_ep_am_pending)
dcs_dci_leak is not relevant anymore